ecf/scripts/gdas/wave/post/jgdas_wave_postsbs.ecf

```diff
@@ -4,8 +4,7 @@
 #PBS -q %QUEUE%
 #PBS -A %PROJ%-%PROJENVIR%
 #PBS -l walltime=00:20:00
-#PBS -l place=vscatter,select=1:mpiprocs=128:ompthreads=1:ncpus=128
-#PBS -l place=excl
+#PBS -l place=vscatter,select=1:mpiprocs=8:ompthreads=1:ncpus=8
```

Comment:
excl or memory? Don't know it's memory needs though.

Reply:
1GB for mem

Reply:
Resolved in a1ca249a